packages/create-app/templates/next-app/packages/app/src/plugins/home/index.tsx

```diff
@@ -0,0 +1,8 @@
+
+import { createFrontendPlugin } from '@backstage/frontend-plugin-api';
+import { HomePage } from './HomePage';
+
+export const homePlugin = createFrontendPlugin({
```

Comment:
I wonder if this would be better as a module instead?

```ts
createFrontendModule({
  pluginId: 'app',
  moduleId: 'home-redirect'
  extensions: [HomeRedirect]
})
```

I think it's just a little confusing to have a `home` plugin already when one exists but it just doesn't work yet.

Reply:
I tried to use module before, and it doesn't work. Unless I'm doing something wrong, I think it only overrides a page if a plugin with the plugin id actually exist